### PR TITLE
Move from image_generation python conversion scripts to optimum-cli

### PR DESCRIPTION
**Pull request by yatarkan**
CVS-132863

### CODE REVIEW

File: image_generation/stable_diffusion_1_5/cpp/README.md [hunk 1]
```diff
@@ -25,31 +29,29 @@ conda install openvino c-compiler cxx-compiler make
...
+python -m pip install -r requirements.txt
 python -m pip install ../../../thirdparty/openvino_tokenizers/[transformers]
 ```
 2. Download a huggingface SD v1.5 model like:
 - [runwayml/stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5)
 - [dreamlike-anime-1.0](https://huggingface.co/dreamlike-art/dreamlike-anime-1.0) to run Stable Diffusion with LoRA adapters.
 
+   Example command for downloading and exporting FP16 model:
+   ```shell
+   export LD_LIBRARY_PATH="$CONDA_PREFIX/lib"
```
[the remark this answers is not in the document]
Comment (ilya-lavrenov) [reply image_generation/stable_diffusion_1_5/cpp/README.md:41]: let's use this approach https://github.com/openvinotoolkit/openvino/pull/23897

[the remark this answers is not in the document]
Comment (yatarkan) [reply image_generation/stable_diffusion_1_5/cpp/README.md]: Fixed

File: image_generation/stable_diffusion_1_5/cpp/src/main.cpp [hunk 2]
```diff
@@ -70,13 +77,78 @@ void apply_lora(std::shared_ptr<ov::Model> model, InsertLoRA::LoRAMap& lora_map)
     }
 }
 
-StableDiffusionModels compile_models(const std::string& model_path, const std::string& device,
-                                     const std::string& lora_path, const float alpha, const bool use_cache) {
+void reshape_text_encoder(std::shared_ptr<ov::Model> model, size_t batch_size, size_t tokenizer_model_max_length) {
+    ov::PartialShape input_shape = model->input(0).get_partial_shape();
+    input_shape[0] = batch_size;
+    input_shape[1] = tokenizer_model_max_length;
+    std::map<std::string, ov::PartialShape> name_to_shape{{model->input(0).get_any_name(), input_shape}};
```
[the remark this answers is not in the document]
Comment (yatarkan) [reply image_generation/stable_diffusion_1_5/cpp/src/main.cpp]: Fixed

File: .github/workflows/stable_diffusion_1_5_cpp.yml [hunk 3]
```diff
@@ -1,84 +1,121 @@
...
           cmake --build ./build/ --config Release --parallel
+
       - name: Run app
+        working-directory: ${{ env.working_directory }}
         run: |
-          set -e
-          source ./openvino/setupvars.sh
-          cd ./image_generation/stable_diffusion_1_5/cpp/build
-          ./stable_diffusion -m ../scripts/runwayml/stable-diffusion-v1-5 -t FP16_dyn
+          ./build/stable_diffusion -m ./models/stable_diffusion_v1_5_ov -t FP16 --dynamic
```
[the remark this answers is not in the document]
Comment (yatarkan) [reply .github/workflows/stable_diffusion_1_5_cpp.yml]: Fixed

File: image_generation/stable_diffusion_1_5/cpp/src/main.cpp [hunk 4]
```diff
@@ -263,14 +372,30 @@ int32_t main(int32_t argc, char* argv[]) try {
...
+                                                  height,
+                                                  width,
+                                                  num_images);
     ov::InferRequest unet_infer_request = models.unet.create_infer_request();
 
     ov::PartialShape sample_shape = models.unet.input("sample").get_partial_shape();
-    OPENVINO_ASSERT(sample_shape.is_dynamic() || (sample_shape[2] * 8 == width && sample_shape[3] * 8 == height),
-        "UNet model has static shapes [1, 4, H/8, W/8] or dynamic shapes [?, 4, ?, ?]");
+    OPENVINO_ASSERT(sample_shape.is_dynamic() || (sample_shape[2] * 8 == height && sample_shape[3] * 8 == width),
+                    "UNet model has static shapes [1, 4, H/8, W/8] or dynamic shapes [?, 4, ?, ?]");
```
[the remark this answers is not in the document]
Comment (yatarkan) [reply image_generation/stable_diffusion_1_5/cpp/src/main.cpp:388]: Fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**